lib/view.js

```diff
@@ -199,10 +222,11 @@ class TerminusView extends View {
 
     this.ptyProcess.on('terminus:title', title => {
       this.process = title
+      this.emit('did-change-title')
     })
-    this.terminal.on('title', title => {
-      this.title = title
-    })
+    // this.terminal.on('title', title => {
+    //   this.title = title
+    // })
```

Comment:
I'll be interested to check these title changes out. =)

Reply:
Good NewsI These changes you made into title actually fix #12

Reply:
Actually only fixed Linux side, Windows now doesnt even show any other name but xterm-256color